### ISSUE TITLE
Soma FM not working

### ISSUE THREAD

**Issue by inverse** [berrberr/streamkeys#251]
For some reason Soma FM is not working - Do you know why? An example URL is:

http://somafm.com/player/#/now-playing/defcon

Do we need to be explicit about the controller? or is what we got enough:

```
      "somafm": { name: "SomaFM", url: "http://somafm.com" },
```

**Comment by berrberr** [comment]
Fixed in #262 :)

**Comment by inverse** [comment]
Super Awesome
